pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -199,6 +199,13 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 				node.Name, task.Namespace, task.Name)
 		}
 
+		// Predicate checks whether enough resource is available for the task in that node
+		clonedNode := node.Idle.Clone()
+		if !task.InitResreq.LessEqual(clonedNode.Add(node.Releasing)) {
+			return fmt.Errorf("task <%s/%s> ResourceFit failed on node <%s>",
+				node.Name, task.Namespace, node.Name)
```

Comment:
should be `task.Namespace, task.Name, node.Name`.

Reply:
done